### PR TITLE
Mark 403 (Forbidden) for fetching turn credentials as terminal error and avoid retrying

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:**
Mark 403 (Forbidden) for fetching turn credentials as terminal error and avoid retrying

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Tested manually & added unit test cases.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aburnson** [comment]
Why are there so many changes in this PR? It looks like a lot of unnecessary copyright year bumps.

**Comment by anuranduttaroy** [comment]
> Why are there so many changes in this PR? It looks like a lot of unnecessary copyright year bumps.

It was complaining about the copyright statement in the files I had touched. So thought of updating all of them.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Disable audio capture for Electron Screen Capture
+- Mark 403 (Forbidden) for fetching turn credentials as terminal error and avoid retrying.
```
Comment (XHatan) [inline CHANGELOG.md:23]: Could you move this to Unrelease section ?

**Comment by anuranduttaroy** [comment]
Will send a new PR without the license changes.

**Pull request closed without merging**